addon/templates/components/frost-list-item-content.hbs

```diff
@@ -0,0 +1,55 @@
+<div class={{concat
+    'frost-list-item-container'
+    (if (eq index 0) ' first')
+    (if (eq index (sub _items.length 1)) ' last')
+    (if model.states.isSelected ' is-selected')
+    (if (and isAnyTypedItemExpansion (not itemExpansion)) ' is-any-expansion')
```

Comment:
Let's make sure that only `isAnyTypedItemExpansion` is check here. I don't think we need itemExpansion. (this take in consideration that `isAnyTypedItemExpansion` will check `itemExpansion`)

Reply:
I've changed `isAnyTypedItemExpansion` to `isAnyItemExpansion` but we will still need the `(not itemExpansion)` to account for cases when there is expansion on other list items but not the current list item.